graphql/src/main/java/com/scalar/db/graphql/server/ServerConfig.java

```diff
@@ -0,0 +1,110 @@
+package com.scalar.db.graphql.server;
+
+import com.google.common.base.Strings;
+import edu.umd.cs.findbugs.annotations.SuppressFBWarnings;
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.Arrays;
+import java.util.LinkedHashSet;
+import java.util.Properties;
+import java.util.Set;
+import javax.annotation.concurrent.Immutable;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+@Immutable
+@SuppressFBWarnings("JCIP_FIELD_ISNT_FINAL_IN_IMMUTABLE_CLASS")
+class ServerConfig {
+  public static final String PREFIX = "scalar.db.graphql.";
+  public static final String PORT = PREFIX + "port";
+  public static final String PATH = PREFIX + "path";
+  public static final String NAMESPACES = PREFIX + "namespaces";
+  public static final String GRAPHIQL = PREFIX + "graphiql";
+
+  public static final int DEFAULT_PORT = 8080;
+  public static final boolean DEFAULT_GRAPHIQL = true;
+  private static final String DEFAULT_PATH = "/graphql";
+  private static final Logger LOGGER = LoggerFactory.getLogger(ServerConfig.class);
+  private final Properties props;
+  private int port;
+  private String path;
+  private Set<String> namespaces;
+  private boolean graphiql;
+
+  public ServerConfig(File propertiesFile) throws IOException {
+    this(new FileInputStream(propertiesFile));
```

Comment:
I found we forget to close this stream:
```suggestion
    try (FileInputStream stream = new FileInputStream(propertiesFile)) {
      props = new Properties();
      props.load(stream);
    }
    load();
```

I will fix the other `XXXConfig` classes for the same thing.